fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/DefaultLoanLifecycleStateMachine.java

```diff
@@ -126,6 +126,11 @@ private LoanStatus getNextState(LoanEvent loanEvent, Loan loan) {
                     newState = overpaidTransition();
                 }
             break;
+            case LOAN_CHARGEBACK:
+                if (anyOfAllowedWhenComingFrom(from, LoanStatus.CLOSED_OBLIGATIONS_MET, LoanStatus.OVERPAID, LoanStatus.ACTIVE)) {
+                    newState = activeTransition();
```

Comment:
@josehernandezfintecheandomx Please change it to set the "from" state to the "newState" AS-IS till the loan balance calculation and repayment schedule logic got implemented for the CHARGEBACK

Reply:
Done! Remove